main() {
    try {
        var x = 1;
    } catch (ex) {
    }
}
